src/Symfony/Component/Uid/AbstractUid.php

```diff
@@ -164,6 +164,11 @@ public function compare(self $other): int
         return (\strlen($this->uid) - \strlen($other->uid)) ?: ($this->uid <=> $other->uid);
     }
 
+    public function toString(): string
+    {
+        return $this->uid;
```

Comment:
to make it easier to decorate if needed?
```suggestion
        return $this->__toString();
```